avocado/core/suite.py

```diff
@@ -62,9 +63,24 @@ def _convert_to_dry_run(self):
     def _from_config_with_loader(cls, config, name=None):
         references = config.get('run.references')
         ignore_missing = config.get('run.ignore_missing_references')
+        verbose = config.get('core.verbose')
+        subcommand = config.get('subcommand')
+
+        # To-be-removed: For some reason, avocado list will display more tests
+        # if in verbose mode. IMO, this is a little inconsistent with the 'run'
+        # command.  This hack was needed to make one specific test happy.
+        tests_mode = DiscoverMode.DEFAULT
+        if subcommand == 'list':
```

Comment:
This raised my eye browns... The list command should ask the suite to act this way, and not the suite looking up whether it's being used by a specific command IMO.

Reply:
Yes, I thought about that too. But to avoid changing the API and since this discover mode will be removed soon, maybe we could accept this.

What do you think? Do you still think that is valid the change?